### DISCUSSION TITLE
Typing in all caps completes to lowercase

### DISCUSSION THREAD

**Discussion by steven-cutting** [company-mode/company-mode#1187] [Q&A] [answered]
Ex: When typing the word 'SPECIFIC' in all caps in a comment (haven't tested this out side of an elsip comment), it will auto complete to something like 'SPECIfic'. 

Basically the auto completed portion of the word does not match the case of the rest of the word.

I've only tested this in the comments of elisp files.

Let me know if I can provide any more info.

This isn't that big of a deal, but it would be nice.

**Comment by dgutov** [comment, marked as answer]
The relevant variable to customize is `company-dabbrev-downcase`.

**Comment by steven-cutting** [comment]
Ah, thank you.
